.github/workflows/pytest.yml

```diff
@@ -12,7 +12,6 @@ jobs:
     strategy:
       matrix:
         python-version:
-        - 3.6
```

Comment:
remove 3.7 and add:

        - "3.8"
        - "3.9"
        - "3.10"
        - "3.11"